src/ru/skypro/Main.java

```diff
@@ -1,8 +1,92 @@
 package ru.skypro;
 
+import java.util.Arrays;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+        task1();
+        task2();
+        task3();
+//   task4();
+        task5();
+        task6();
+    }
+
+
+    private static void task1() {
+        int[] monthExpense = {155_002, 100_670, 141_000, 163_000, 105_707, 140_000, 150_000, 125_000, 100_087, 100_450, 168_777, 100_000, 100_052, 108_000, 100_065, 100_047, 104_000, 100_007, 100_050, 136_057, 100_000, 100_000, 100_000, 100_000, 100_350, 100_520, 108_000, 189_000, 200_000, 107_000};
+        int sum = 0;
+        for (int i = 0; i < monthExpense.length; i++) {
+            sum = sum + monthExpense[i];
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей.");
+    }
+
+    private static void task2() {
+        int[] monthExpense = {155_002, 100_670, 141_000, 163_000, 105_707, 140_000, 150_000, 125_000, 100_087, 100_450, 168_777, 100_000, 100_052, 108_000, 100_065, 100_047, 104_000, 100_007, 100_050, 136_057, 100_000, 100_000, 100_000, 100_000, 100_350, 100_520, 108_000, 189_000, 200_000, 107_000};
+        int smallest = monthExpense[0];
+        int largest = monthExpense[0];
+        for (int i = 0; i <= 29; i++) {
```

Comment:
```suggestion
        for (int i = 0; i <= monthExpense.length; i++) {
```
чтобы не было соблазна каждый раз пересчитывать вручную количество элементов в массиве

Reply:
точно! но прописывая значение 29, мне было прощепонять прописаны ли все 30 заложенных значений в массив или я ошиблась и записала больше/меньше. Согласно условиям задачи, необходимо было посчитать данные за 30 дней. А так как данные варьируются от 100 000 до 200 000 + вбиваются вручную, я и использовала 29 вместо monthExpense.length.
Но возможно в этом случае мне было бы проще прописать значение каждого индекса массива как
int[] monthExpense = new int[30]
monthExpense[0] = 155_002;
monthExpense[1] = 100_670; и тд.
а затем как раз и использовать monthExpense.length.

Но это выглядит для меня более очевидным и понzтным сейчас, после субботнего вебинара.

Спасибо!